src/Events/Admin/Onboarding/Data.php

```diff
@@ -0,0 +1,761 @@
+			],
+			'nzd'     => [
+				'code'   => 'NZD',
+				'name'   => __( 'New Zealand Dollar', 'the-events-calendar' ),
+				'symbol' => '$',
+				'entity' => '&#36;',
+			],
+			'nok'     => [
+				'code'   => 'NOK',
+				'name'   => __( 'Norwegian Krone', 'the-events-calendar' ),
+				'symbol' => 'kr',
+				'entity' => 'kr',
+			],
+			'php'     => [
+				'code'   => 'PHP',
+				'name'   => __( 'Philippine Peso', 'the-events-calendar' ),
+				'symbol' => '₱',
+				'entity' => '&#x20B1;',
+			],
+			'pln'     => [
+				'code'   => 'PLN',
+				'name'   => __( 'Polish Złoty', 'the-events-calendar' ),
+				'symbol' => 'zł',
+				'entity' => 'z&#x142;',
+			],
+			'rub'     => [
+				'code'   => 'RUB',
+				'name'   => __( 'Russian Ruble', 'the-events-calendar' ),
+				'symbol' => '₽',
+				'entity' => '&#8381;',
+			],
+			'sek'     => [
+				'code'   => 'SEK',
+				'name'   => __( 'Swedish Krona', 'the-events-calendar' ),
+				'symbol' => 'kr',
+				'entity' => 'kr',
+			],
+			'sgd'     => [
+				'code'   => 'SGD',
+				'name'   => __( 'Singapore Dollar', 'the-events-calendar' ),
+				'symbol' => '$',
+				'entity' => '&#36;',
+			],
+			'zar'     => [
+				'code'   => 'ZAR',
+				'name'   => __( 'South African Rand', 'the-events-calendar' ),
+				'symbol' => 'R',
+				'entity' => 'R',
+			],
+			'chf'     => [
+				'code'   => 'CHF',
+				'name'   => __( 'Swiss Franc', 'the-events-calendar' ),
+				'symbol' => 'Fr',
+				'entity' => 'Fr',
+			],
+			'twd'     => [
+				'code'   => 'TWD',
+				'name'   => __( 'Taiwan New Dollar', 'the-events-calendar' ),
+				'symbol' => '$',
+				'entity' => '&#36;',
+			],
+			'thb'     => [
+				'code'   => 'THB',
+				'name'   => __( 'Thai Baht', 'the-events-calendar' ),
+				'symbol' => '฿',
+				'entity' => '&#x0E3F;',
+			],
+			'trl'     => [
+				'code'   => 'TRL',
+				'name'   => __( 'Turkish Lira', 'the-events-calendar' ),
+				'symbol' => '₺',
+				'entity' => '&#8378;',
+			],
+			'usd'     => [
+				'code'   => 'USD',
+				'name'   => __( 'US Dollar', 'the-events-calendar' ),
+				'symbol' => '$',
+				'entity' => '&#36;',
+			],
+			'usdcent' => [
+				'code'   => 'USDCENT',
+				'name'   => __( 'US Cent', 'the-events-calendar' ),
+				'symbol' => '¢',
+				'entity' => '&cent;',
+			],
+			'vnd'     => [
+				'code'   => 'VND',
+				'name'   => __( 'Vietnamese Dong', 'the-events-calendar' ),
+				'symbol' => '₫',
+				'entity' => '&#8363;',
+			],
+		];
+
+		return (array) apply_filters( 'tec_events_onboarding_wizard_currencies_list', $default_currencies );
+	}
+
+	/**
+	 * Get the saved wizard settings.
+	 *
+	 * @since 7.0.0
+	 *
+	 * @return array
+	 */
+	public function get_wizard_settings() {
+		return get_option( 'tec_onboarding_wizard_data', [] );
+	}
+
+	/**
+	 * Update the wizard settings.
+	 *
+	 * @since 7.0.0
+	 *
+	 * @param array $settings The settings to update.
+	 */
+	public function update_wizard_settings( $settings ) {
+		update_option( 'tec_onboarding_wizard_data', $settings );
```

Comment:
No return ? Its fine whether it succeeds or fails ?

Reply:
It's not necessary for the user's experience, but we can still record/send the message.